instructions.md

```diff
@@ -0,0 +1,105 @@
+## Authorize Github
+
+ - Go to sandbox.fireflyblockchain.com
+ - At the top right, click on "login"
+ - At the Github OAuth screen, click on the "Authorize rv-jenkins" button
+
+![](https://raw.githubusercontent.com/runtimeverification/firefly-demo/guy_doc/img/1authorize.png)
+
+## Set up an access token
+
+ - After being redirected, go to the "Products" drop-down in the top right and click on "test runner"
+
+![](https://raw.githubusercontent.com/runtimeverification/firefly-demo/guy_doc/img/2testrunner.png)
+ - Click on the "install now" button at the new page
+ - Type in a name for your new token and click on "create"
+
+![](https://raw.githubusercontent.com/runtimeverification/firefly-demo/guy_doc/img/3createtoken.png)
+ - Save the token for later
+
+## Set up CI
+
+In your CI script, have a stage that does the following:
+
+Install the Firefly client. The following dependencies must be installed for Ubuntu:
+```
+autoconf
+bison
+clang-8
+cmake
+curl
+flex
+gcc
+jq
+libboost-test-dev
+libcrypto++-dev
+libffi-dev
+libgflags-dev
+libjemalloc-dev
+libmpfr-dev
+libprocps-dev
+libsecp256k1-dev
+libssl-dev
+libtool
+libyaml-dev
+lld-8
+llvm-8-tools
+make
+maven
+netcat-openbsd
+opam
+openjdk-11-jdk
+pandoc
+pkg-config
+python3
+rapidjson-dev
+software-properties-common
+time
+zip
+zlib1g-dev
+```
```

Comment:
Double check that this list is only the things left in `Dockerfile`.

At least `opam` is still here.